src/types.ts

```diff
@@ -12,10 +12,39 @@ export interface WrapperMap {
     readonly timestamp: string;
     readonly version: Version;
     readonly wrappers: {
-        button: WrapperPair[];
-        unbutton: WrapperPair[];
+        [key: string]: WrapperPair[];
     };
     readonly keywords?: string[];
     readonly tags?: Tags;
     readonly logoURI?: string;
+    readonly tokens?: TokenData[];
+}
+
+export interface TokenConfig {
+    name: string;
+    symbol: string;
+    decimals: number;
+    primaryChainId: number;
+    chains: {
+        [key: string]: {
+            address: string;
```

Comment:
Chain specific data can include more properties if required.
One example where this occurs is for USDT, the testnet contract has a different decimals value